src/main/java/io/pivotal/cla/service/github/MylynGitHubApi.java

```diff
@@ -65,18 +69,28 @@
 import lombok.Data;
 import lombok.SneakyThrows;
 
+/**
+ * @author Rob Winch
+ * @author Mark Paluch
+ */
 @Component
 public class MylynGitHubApi implements GitHubApi {
+
 	private static final String AUTHORIZE_URI = "login/oauth/access_token";
 	public final static String CONTRIBUTING_FILE = "CONTRIBUTING";
 	public final static String ADMIN_MAIL_SUFFIX = "@pivotal.io";
 	public final static Pattern PULL_REQUEST_CALLBACK_PATTERN = Pattern.compile(".*" + UrlBuilder.pullRequestHookCallbackPath("") + "([a-zA-Z0-9\\-\\s\\%\\+]*)(\\?.*)?");
 
-	ClaOAuthConfig oauthConfig;
-
-	String authorizeUrl;
+	public final static String CONTRIBUTOR_LICENSE_AGREEMENT = "Contributor License Agreement";
+	public final static String OBVIOUS_FIX_CLA_NOT_REQUIRED = String.format(
+			"This Pull Request contains an obvious fix. Signing the %s not necessary.", CONTRIBUTOR_LICENSE_AGREEMENT);
+	public final static String THANK_YOU = "Thank you for signing the";
+	public final static String PLEASE_SIGN = "Please sign the";
+	public final static String OBVIOUS_FIX = "obvious fix";
 
-	RestTemplate rest = new RestTemplate();
+	private ClaOAuthConfig oauthConfig;
+	private String authorizeUrl;
+	private  RestTemplate rest = new RestTemplate();
```

Comment:
There's no reason why the fields of `MylynGitHubApi` should be visible to other classes.